lib/periodic_jobs.rb

```diff
@@ -207,7 +207,7 @@
   mgr.register('0 2,9,16 * * 1-5', 'VBADocuments::FlipperStatusAlert')
 
   # Rotates Lockbox/KMS record keys and _ciphertext fields every October 12th (when the KMS key auto-rotate)
-  mgr.register('0 3 * * *', 'KmsKeyRotation::BatchInitiatorJob')
+  mgr.register('10 1 * * *', 'KmsKeyRotation::BatchInitiatorJob')
```

Comment:
Wait... this runs every night. Is it only supposed to run on Oct 12?

Reply:
Yeah every night because it can pick up records that had issues with rotation, decryption, etc

Reply:
Does the comment on line 209 still apply?

Reply:
Yep still relevant :) It runs daily but the major run with occur on the 12th every year after the key is rotated to encrypt the fields with the new key material.